decidim-accountability/.rubocop.yml

```diff
@@ -0,0 +1 @@
+../.rubocop-rails.yml
```

Comment:
Oh, that's a clever hack. But right now we're running rubocop from the root folder, will this config be applied still? Or should we `cd` into each project in order to run rubocop so this config gets applied?